### ISSUE TITLE
False positives for 932260 with cron

### ISSUE THREAD

**Issue by superlgn** [coreruleset/coreruleset#3577]
### Description

I'm seeing another false positive on rule 932260, with a last name of "Cronk" matching against unix-shell-4andup.ra:cron.

### How to reproduce the misbehavior (-> curl call)

The sandbox isn't working for me at the moment (curl: (52) Empty reply from server), but this should trigger it:

```
curl -H "x-format-output: txt-matched-rules" https://sandbox.coreruleset.org/get?last_name=Cronk
```

And the fix, I believe, should be similar to the changes in issue #3490, adding a @ after the .ra entry for 'cron', eg 'cron@'.

I'll open a pull request in a minute here.

### Logs

``
[Tue Feb 20 09:15:37.664128 2024] [:error] [pid 32733] [client a.b.c.d:57301] [client a.b.c.d] ModSecurity: Access denied with code 403 (phase 2). Pattern match "(?i)(?:^|=)[\\\\s\\\\v]*(?:t[\\"'\\\\)\\\\[-\\\\x5c]*(?:(?:(?:\\\\|\\\\||&&)[\\\\s\\\\v]*)?\\\\$[!#\\\\(\\\\*\\\\-0-9\\\\?-@_a-\\\\{]*)?\\\\x5c?i[\\"'\\\\)\\\\[-\\\\x5c]*(?:(?:(?:\\\\|\\\\||&&)[\\\\s\\\\v]*)?\\\\$[!#\\\\(\\\\*\\\\-0-9\\\\?-@_a-\\\\{]*)?\\\\x5c?m[\\"'\\\\)\\\\[-\\\\x5c]*(?:(?:(?:\\\\|\\\\||&&)[\\\\s\\\\v]*)?\\\\$[ ..." at ARGS:last_name. [file ".../crs/rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf"] [line "498"] [id "932260"] [msg "Remote Command Execution: Direct Unix Command Execution"] [data "Matched Data: CRON found within ARGS:last_name: CRONK"] [severity "CRITICAL"] [ver "OWASP_CRS/4.0.0-rc2"] [tag "application-multi"] [tag "language-shell"] [tag "platform-unix"] [tag "attack-rce"] [tag "paranoia-level/1"] [tag "OWASP_CRS"] [tag "capec/1000/152/248/88"] [tag "PCI/6.5.2"] [hostname "www.example.com"] [uri "/.../"] [unique_id "ZdTCGcLU6cSJGsumMPMFWwAAAAw"], referer: https://www.example.com/.../
``

### Your Environment

* CRS version (e.g., v3.3.4): 4.0.0-rc2
* Paranoia level setting (e.g. PL1) : PL1
* ModSecurity version (e.g., 2.9.6): 2.9.3
* Web Server and version or cloud provider / CDN (e.g., Apache httpd 2.4.54): 2.4.38
* Operating System and version: Debian 10

### Confirmation

[x] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by azurit** [comment]
@superlgn Can you confirm that this problem is fixed? Thank you.

**Comment by superlgn** [comment]
Yes, this is fixed on main. Thanks.
